crates/router/src/routes/user.rs

```diff
@@ -350,6 +350,23 @@ pub async fn invite_user(
     ))
     .await
 }
+pub async fn invite_multiple_user(
+    state: web::Data<AppState>,
+    req: HttpRequest,
+    payload: web::Json<Vec<user_api::InviteUserRequest>>,
+) -> HttpResponse {
+    let flow = Flow::InviteUser;
```

Comment:
Should we create a different flow for this?

Reply:
Later we will be using this api only after deprecating the previous version. I guess this flow is fine.

Reply:
yes we should.